style.css

```diff
@@ -0,0 +1,448 @@
+:root {
+    --primary-color: powderblue;
+    --active: green;
+    --inactive: grey;
+
+}
+body {
+    font-family: 'Montserrat',
+    sans-serif;
+    background-color: palegreen;
+    max-width: 900px;
```

Comment:
it's a good practice to use relative units while defining the width/height property instead of using px(absolute units).